File: Components/WinForms/CS/SpreadsheetMainDemo/Modules/OutlineGroupingSubtotal.cs
using DevExpress.XtraBars.Ribbon;

namespace DevExpress.XtraSpreadsheet.Demos
{
    public partial class OutlineGroupingSubtotal : SpreadsheetRibbonTutorialControlBase
    {

        public OutlineGroupingSubtotal()
        {
            InitializeComponent();
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("OutlineGrouping_template.xlsx"));
        }

        #region Properties
        protected internal override RibbonPage SelectedRibbonPage { get { return dataRibbonPage1; } }
        #endregion
    }
}

File: Components/WinForms/VB/SpreadsheetMainDemo/Modules/OutlineGroupingSubtotal.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraBars.Ribbon

Namespace DevExpress.XtraSpreadsheet.Demos
	Partial Public Class OutlineGroupingSubtotal
		Inherits SpreadsheetRibbonTutorialControlBase

		Public Sub New()
			InitializeComponent()
			spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("OutlineGrouping_template.xlsx"))
		End Sub

		#Region "Properties"
		Protected Friend Overrides ReadOnly Property SelectedRibbonPage() As RibbonPage
			Get
				Return dataRibbonPage1
			End Get
		End Property
		#End Region
	End Class
End Namespace
